frontend/smart-kickers-game/src/components/Heatmap/Heatmap.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import HeatMap from 'react-heatmap-grid';
+import './Heatmap.css';
+import { Colors } from './Colors';
+import useHeatmap from '../../hooks/useHeatmap';
+
+function Heatmap() {
+  const [{ loading, error, heatmap }] = useHeatmap();
+  if (loading) return <div>Loading...</div>;
+  if (error) return <div>Error</div>;
+
+  function chooseColor(value) {
+    let chosenColor;
+    switch (true) {
+      case value <= 2:
+        chosenColor = Colors.none;
+        break;
+      case value <= 5:
+        chosenColor = Colors.blue;
+        break;
+      case value <= 10:
+        chosenColor = Colors.purple;
+        break;
+      case value <= 15:
+        chosenColor = Colors.green;
+        break;
+      case value <= 25:
+        chosenColor = Colors.yellow;
+        break;
+      case value > 25:
+        chosenColor = Colors.red;
+        break;
+      default:
+        chosenColor = Colors.none;
+        break;
+    }
+    return `rgba(${chosenColor.red}, ${chosenColor.green},${chosenColor.blue},  ${chosenColor.opacity} )`;
```

Comment:
What is more it can be move outside of the component because it doesn't use any inner state. `Colors.js` can be a good place where you keep the all the colors and the function that uses them.

Reply:
Done